www/install/sql/centreon/Update-DB-21.10.2.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM `cb_tag` where `tagname` = 'logger';
```

Comment:
your target is develop, so you should manage upgrade in 22.04.0-beta.1 script
Furthermore, could you add this in a php upgrade script instead of sql ? It will allow to manage exceptions / logs